sdk/websocket.go

```diff
@@ -26,6 +26,10 @@ var (
 	confirmedAddedChannels     = make(map[string]chan Transaction)
 )
 
+type subscribeInfo struct {
```

Comment:
PLease, move private types definitions after public

Reply:
solved in the websocket branch